components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix.F

```diff
@@ -95,7 +95,7 @@ subroutine ocn_tracer_hmix_tend(meshPool, layerThicknessEdge, layerThickness, zM
          RediKappa
 
       real (kind=RKIND), dimension(:,:), intent(in) :: &
-         layerThicknessEdge, &!< Input: thickness at edge
+         layerThicknessEdge, &!< Input: flux-related thickness at edge
```

Comment:
Note that there are some instances (such as this one) in which I merely noted which `layerThicknessEdge` should be supplied as inputs to routines, and provided that input when the routine is called, rather than changing the variable name to `layerThicknessEdgeFlux` or `layerThicknessEdgeCenter` everywhere. If preferred, I could do the latter instead.

Reply:
I think it would make searching the code easier if these were all changed to `layerThicknessEdgeFlux` or `layerThicknessEdgeCenter`(`ed`).  But it's not a strong opinion.